src/dom-to-image.js

```diff
@@ -6,6 +6,14 @@
     var fontFaces = newFontFaces();
     var images = newImages();
 
+    // Default impl options
+    var defaultOptions = {
+        // Default placeholder is a 1x1 gray square
+        placeholder: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAMSURBVBhXY7h79y4ABTICmGnXPbMAAAAASUVORK5CYII=",
```

Comment:
If I'm not mistaken, browsers already draw placeholders in place of broken images. I don't want to confuse anyone who didn't ask for a non-default image placeholder with a grey pixel they will likely not notice.